### PR TITLE
OPS: Update to RN 67

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: .bundle/config [hunk 1]
```diff
@@ -0,0 +1,2 @@
+BUNDLE_PATH: "vendor/bundle"
```
Comment (Overtorment) [inline .bundle/config:1]: this is new. whats this?

File: .gitattributes [hunk 2]
```diff
@@ -1,5 +0,0 @@
-*.pbxproj -text
-*.patch -text
-# Windows files should use crlf line endings
-# https://help.github.com/articles/dealing-with-line-endings/
-*.bat text eol=crlf
```
Comment (Overtorment) [inline .gitattributes:5]: i think we need this file, or developers on ms windows have issues. is that ok that this file is being deleted?

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2tkm0e8b1a4gta8kyv8ypj2030?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9517)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7924hgmr76ertd474qecygwt34?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9519)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/w905u262y0uhmrz2pah4gqz1kg?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9522)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/k9h10hhn999e7hrj5envavmnxm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9523)

**Comment by Overtorment** [comment]
is this a WIP  PR..?

**Comment by marcosrdz** [comment]
Yup. Waiting on official new Tor release. The PR we were waiting for got merged but no tag has been made

**Comment by Overtorment** [comment]
@gabidi  humbly waiting for a release 🙏🏻

**Comment by gabidi** [comment]
Published :)

**Comment by marcosrdz** [comment]
> Published :)

Just saw! Thank you!

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xv6p94f5wajdcavfg1npr943h4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9534)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6z9knb987zmc1w5n4yyqc0zr50?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9539)

File: .gitattributes (hunk 2 above)
Comment (marcosrdz) [reply .gitattributes:5]: It was removed on 0.67

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+2.7.4
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply .ruby-version:1]: Yup. Following the upgrade helper RN guide.

https://react-native-community.github.io/upgrade-helper/

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply Gemfile:1]: Yup. Following the upgrade helper RN guide.

https://react-native-community.github.io/upgrade-helper/

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/v4ry9dngu5dxm9gu6y4m667fg8?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9558)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Pull request merged**